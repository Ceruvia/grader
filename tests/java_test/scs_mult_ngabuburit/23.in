4 3 7 1 3 2 3 2 2 0 3 4 3 0 1 0 2 