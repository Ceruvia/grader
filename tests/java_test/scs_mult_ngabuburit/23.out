Pergi ke luar rumah!
Pergi ke luar rumah!
Berlari di gasibu sejauh 10 km!
Pergi ke luar rumah!
Berlari di gasibu sejauh 10 km!
Pergi ke luar rumah!
Berlari di gasibu sejauh 10 km!
Membeli kolak!
Sudah jenuh! Lakukan aktivitas lain
